builtin/logical/pki/path_issue_sign.go

```diff
@@ -224,6 +224,8 @@ func (b *backend) pathIssueSignCert(ctx context.Context, req *logical.Request, d
 			return logical.ErrorResponse(err.Error()), nil
 		case errutil.InternalError:
 			return nil, err
+		default:
```

Comment:
No panics please.

Reply:
In that case, please fix it to something else :-)

Reply:
I don't understand what you're looking for.  Should I wrap an unexpected error type with an InternalError?